lib/dialects/sqlite/query-generator.js

```diff
@@ -332,7 +332,7 @@ const QueryGenerator = {
 
   quoteIdentifier(identifier) {
     if (identifier === '*') return identifier;
-    return Utils.addTicks(identifier, '`');
+    return Utils.addTicks(Utils.removeTicks(identifier, '`'), '`');
```

Comment:
@felixfbecker After more consideration, we should not escape identifiers and just remove them.  Supporting identifiers with identifying characters in them seems silly.